packet/static/js/signing-0.js

```diff
@@ -13,7 +13,7 @@ $(document).ready(function () {
             .then((willSign) => {
                 if (willSign) {
                     $.ajax({
```

Comment:
This doesn't have any error handling; if signing fails there will be an uncaught promise rejection and it won't be communicated to the user.

Reply:
Yeah, the UI is quite hacky in places still. I didn't put much work into it in this PR though since we're planning to replace it wholesale.